public/locale/en.json

```
@@ -1237,7 +1237,7 @@
   "encounter_suggestion_edit_disallowed": "Not allowed to switch to this option in edit consultation",
   "encounter_type": "Encounter Type",
   "encounters": "Encounters",
-  "end_date": "End date",
+  "end_date": "End Date",
```

we learnt this in school :) 

```suggestion
  "end_date": "End date",
```